main.py

```diff
@@ -0,0 +1,3 @@
+import tcod
+
+print("Hello World!")
```

Comment:
put hello world into a `main` method and call it when we explicitly run the script, using `python main.py`